modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSecondaryIndexTest.java

```diff
@@ -946,23 +968,29 @@ public void testNullsInSearchRow() {
 
             assertQuery("SELECT * FROM t WHERE i1 = ?")
                     .withParams(null)
-                    .matches(containsIndexScan("PUBLIC", "T", "T_IDX"))
+                    .matches(containsTableScan("PUBLIC", "T"))
```

Comment:
there are already numerous tests in ItSecondaryIndexTest with containsTableScan, i think it`s not corresponding to issues this class was intended for and need to be refactored in future.